### PR TITLE
fixed source and target language

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Resources/translations/SonataTimelineBundle.de.xliff [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <file source-language="de" target-language="de" datatype="plaintext" original="file.ext">
```
Comment (Soullivaneuh) [inline Resources/translations/SonataTimelineBundle.de.xliff]: source-language should be `en`, not `de`, isn't it?

ping @rande

Comment (OskarStark) [inline Resources/translations/SonataTimelineBundle.de.xliff]: i looked at ```fr``` and ```es```...

bit confusing :)

Comment (Soullivaneuh) [inline Resources/translations/SonataTimelineBundle.de.xliff]: On the [official doc](http://symfony.com/doc/current/book/translation.html#basic-translation), a french translation sample:

```xml
<!-- messages.fr.xlf -->
<?xml version="1.0"?>
<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
    <file source-language="en" datatype="plaintext" original="file.ext">
        <body>
            <trans-unit id="1">
                <source>Symfony is great</source>
                <target>J'aime Symfony</target>
            </trans-unit>
        </body>
    </file>
</xliff>
```

`source-language` is `en` and `target-language` is not specified at all.

Comment (OskarStark) [inline Resources/translations/SonataTimelineBundle.de.xliff]: https://github.com/sonata-project/SonataTimelineBundle/blob/master/Resources/translations/SonataTimelineBundle.fr.xliff

Comment (Soullivaneuh) [inline Resources/translations/SonataTimelineBundle.de.xliff]: @OskarStark Yes, I trust you don't worry. ;-)

But I think this is not the good way.

Have to investigate.

**Pull request merged**